packages/e2e/test/fixture-projects/sample-project/package.json.template

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "basic-project",
+  "version": "1.0.0",
+  "license": "MIT",
+  "dependencies": {}
+}
```

Comment:
Why is this file needed?

Reply:
I created it because I was receiving this error:
```
Error: cp: no such file or directory: /home/gene/dev/nomiclabs/hardhat/packages/e2e/test/fixture-projects/sample-project/package.json.template
    at Object.error (/home/gene/dev/nomiclabs/hardhat/node_modules/shelljs/src/common.js:110:27)
    at /home/gene/dev/nomiclabs/hardhat/node_modules/shelljs/src/cp.js:246:14
    at Array.forEach (<anonymous>)
    at Object._cp (/home/gene/dev/nomiclabs/hardhat/node_modules/shelljs/src/cp.js:243:11)
    at Object.cp (/home/gene/dev/nomiclabs/hardhat/node_modules/shelljs/src/common.js:384:25)
    at setup (/home/gene/dev/nomiclabs/hardhat/packages/e2e/run-tests.js:70:11)
    at main (/home/gene/dev/nomiclabs/hardhat/packages/e2e/run-tests.js:25:3)
    at Object.<anonymous> (/home/gene/dev/nomiclabs/hardhat/packages/e2e/run-tests.js:96:1)
    at Module._compile (internal/modules/cjs/loader.js:936:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:947:10)
```

Reply:
Ugh, right, I had forgotten about that :sweat_smile: